README.md

```diff
@@ -355,9 +355,55 @@ render status: :forbidden
     Unless they have some meaning in the business domain, don't put methods in
     your model that just format your data (like code generating HTML). These
     methods are most likely going to be called from the view layer only, so their
-    place is in helpers. Keep your models for business logic and data-persistence
+    place is in helpers, if simple, or a separate class if more complex.
+    Keep your models for business logic and data-persistence
     only.
     <sup>[[link](#model-business-logic)]</sup>
+    
+    ```erb
+    <!-- bad (if complex) -->
```

Comment:
An example of non-business logic in a model can be presentation of the price depending on the current user's currency settings, user name/phone number presentation,